hack/ci/e2e.sh

```diff
@@ -168,8 +168,6 @@ run_tests() {
   fi
 
   # ginkgo regexes
-  SKIP="${SKIP:-"Feature|Federation|PerformanceDNS|DualStack|Disruptive|Serial|KubeProxy|GCE|Netpol|NetworkPolicy|256.search.list.characters|LoadBalancer.Service.without.NodePort|type.and.ports.of.a.TCP.service|loadbalancer.source.ranges"}"
-  FOCUS="${FOCUS:-"\\[sig-network\\]"}"
```

Comment:
why?

Reply:
we have now LABEL_FILTER that is the new cool kid on the block https://github.com/kubernetes/test-infra/pull/33075 ... you even can read the filter and understand it 😄